### PR TITLE
Create available-languages.md

### PR DESCRIPTION
**Pull request by gabrielcarneiro97**

### CODE REVIEW

**Comment by gustavo-depaula** [comment]
is this useful? like, a person could just look at the repo's file directory

**Comment by gabrielcarneiro97** [comment]
It can be improved, the repo's file doesn't contain a lot of info that we could put at this file, like the exact number of implementations

File: available-languages.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+15. **GP**
+16. **Haskell**
+17. **Haxe**
+18. **Java**
+19. **JavaScript**
+20. **Kotlin**
+21. **Lua**
+22. **MoonScript**
+23. **OCaml**
+24. **Pearl**
```
Comment (masak) [inline available-languages.md]: typo: "Perl"

**Comment by gustavo-depaula** [comment]
This is too hard to maintain and not that useful. Closing for now.

**Pull request closed without merging**